app/presenters/finder_presenter.rb

```diff
@@ -72,7 +72,19 @@ def email_alert_signup_url
     signup_link = content_item['details']['signup_link']
     return signup_link if signup_link.present?
 
-    email_alert_signup['web_url'] if email_alert_signup
+    filtered_values = facets.each_with_object({}) do |facet, hash|
```

Comment:
maybe better with a reject on has_filters, a map to &value and another bunch of rejects?